### PR TITLE
initial setup for api documentation

### PR DESCRIPTION
**Pull request by bhatiadheeraj**

### CODE REVIEW

File: package-lock.json
Comment (dlevitas) [inline package-lock.json:1]: I'm always nervous about changing this file, tends to cause issues when we update production

Comment (bhatiadheeraj) [reply package-lock.json:1]: No worries ! If we encounter an issue, we will fix it

Comment (dlevitas) [reply package-lock.json:1]: everything else looks good to me

**Comment by dlevitas** [comment]
Looks good

**Pull request merged**